### DISCUSSION TITLE
Is it possible to remap a button to a combo of buttons?  I only see remap options for keyboard or mouse

### DISCUSSION THREAD

**Discussion by ahungry** [AntiMicroX/antimicrox#899] [Q&A]
I have an Xbox Series 2 Elite controller going through xpadneo drivers over bluetooth on GNU/Linux.

This sends the back paddles as distinct events (11 to 14).

I want to remap a back paddle to Left Trigger (any threshold is fine, 32k or w/e) plus a d-pad direction, but I can't see how to do a key remap - I did find controller profiles, but I don't think that would allow me to do this.

Is there a way to do it with antimicrox?  if not, does anyone know if there is a different tool that'd allow it?

Or will I have to make some Rube Goldberg contraption where I use antimicrox to turn the paddle press into a keyboard event, and then a different software to listen to the keyboard event and simulate the joystick press? :joy:

**Comment by BoomerangAide** [comment]
I think that's not the expected use of AntiMicroX.
You're expected to convert all the pad events into keyboard and mouse events.
If you did that, you could just assign to the paddle the keyboard/mouse action of Left Trigger + the keyboard/mouse action of the d-pad direction in Advanced.
